src/Pelagos/Util/DOIutil.php

```diff
@@ -84,68 +90,61 @@ public function generateDoi() : string
      * @param string $url             URL for DOI.
      * @param string $creator         Creator for DOI.
      * @param string $title           Title for DOI.
-     * @param string $publisher       Publisher for DOI.
      * @param string $publicationYear Published Date for DOI.
-     * @param string $status          Status of the DOI, by default is reserved.
+     * @param string $publisher       Publisher for DOI.
      * @param string $resourcetype    Type for DOI Request, by default Dataset.
      *
-     * @throws HttpClientErrorException When there was an 4xx error negotiating with EZID.
-     * @throws HttpServerErrorException When there was an 5xx error negotiating with EZID.
+     * @throws HttpClientErrorException When there was an 4xx error negotiating with REST API.
+     * @throws HttpServerErrorException When there was an 5xx error negotiating with REST API.
      *
-     * @return string The DOI identifier of the DOI either updated or generated.
+     * @return void
      */
     public function createDOI(
         $doi,
         $url,
         $creator,
         $title,
-        $publisher,
         $publicationYear,
-        $status = 'reserved',
+        $publisher = 'Harte Research Institute',
```

Comment:
I liked it better when it was explicitly called. The utility shouldn't really know anything about who it's doing business for.

Reply:
Hmm.. I dont know I thought it is just another variable, which is going to be constant atleast for now. We don't need to explicitly call it

Reply:
Yes, it's a variable but not a constant. In my opinion a utility should be "dumb" and now know anything. Also it's easy to overlook who sets this value. The caller should be responsible for setting it.

Reply:
Okay I can change it back